drivers/devicetree/fdt.c

```diff
@@ -84,3 +85,159 @@ FAR const char *fdt_get(void)
 {
   return g_fdt_base;
 }
+
+/****************************************************************************
+ * Name: fdt_get_irq
+ *
+ * Description:
+ *   Get the interrupt number of the node
+ *
+ ****************************************************************************/
+
+int fdt_get_irq(FAR const void *fdt, int offset, int irqbase)
+{
+  FAR const fdt32_t *pv;
+  int irq = -1;
+
+  pv = fdt_getprop(fdt, offset, "interrupts", NULL);
+  if (pv != NULL)
+    {
+      irq = fdt32_ld(pv + 1) + irqbase;
```

Comment:
In my patch, I've made the +1 offset from pv a parameter passed in, since this offset doesn't seem to be a typical feature.